coinstacks/bitcoin/api/src/controller.ts

```diff
@@ -83,8 +83,8 @@ export class Bitcoin extends Controller implements BaseAPI, BitcoinAPI {
       return {
         pubkey: data.address,
         balance: data.balance,
-        nextReceiveAddressIndex: nextAddressIndexes[0],
-        nextChangeAddressIndex: nextAddressIndexes[1],
+        nextReceiveAddressIndex: nextAddressIndexes[0] || 0,
+        nextChangeAddressIndex: nextAddressIndexes[1] || 0,
```

Comment:
prefer `??` over `||` for default assignment on nullish value
```suggestion
        nextReceiveAddressIndex: nextAddressIndexes[0] ?? 0,
        nextChangeAddressIndex: nextAddressIndexes[1] ?? 0,
```

Reply:
If we are using a default, we may want to construct the return payload checking the `isXpub` value since we will now also be returning address indexes when fetching account details by address which isn't accurate.

Reply:
The other alternative is the client can chose their starting index if undefined